deploy/Chart/templates/rp/deployment.yaml

```diff
@@ -73,8 +73,6 @@ spec:
           mountPath: {{ .Values.global.rootCA.mountPath }}
           readOnly: true
         {{- end }}
-        securityContext:
-          allowPrivilegeEscalation: false
```

Comment:
👍 Thanks for fixing this.

Reply:
For anyone else reviewing this, it's a duplicate of Line 64 in the same file: https://github.com/radius-project/radius/pull/6996/files#diff-92bd08c7c2302e12d2ceb2f7bf0a5bdd0016bb8e2eb2639588e994366e9ee841L64

Reply:
I checked the other files in the chart, and they all look good: https://github.com/search?q=repo%3Aradius-project%2Fradius%20securityContext&type=code